### PR TITLE
[DOCS] New section for AWP docs

### LINKED ISSUE #2623: [DOCS] AWP docs re-org
## Description 

Move the Session view doc, and duplicate the K8s dashboard doc, so that the Cloud Native Security section can have the following structure:

Cloud native security (top-level section)
   - Kubernetes security posture management (subsection)
       - [all the KSPM docs]
 - Cloud workload protection (subsection)
       - Session view
       - K8s dashboard

## Acceptance Test Criteria

Approved by AWP team members and security docs team

### PR DESCRIPTION
**Pull request by benironside**
fixes #2623 

Creates a sub-section "Cloud workload protection" within the "Cloud native security" top level docs section. The new subsection is intended as a home for AWP-related docs, initially including docs for Session view and the K8s dashboard. 

Preview: [Cloud workload protection section](https://security-docs_2629.docs-preview.app.elstc.co/guide/en/security/master/cloud-workload-protection.html)

### CODE REVIEW

File: docs/cloud-native-security/cloud-workload-protection.asciidoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[[cloud-workload-protection]]
+= Cloud workload protection
+
+Cloud workload protection helps you monitor and understand what's happening on your Linux hosts. It uses the Elastic Defend integration to capture Linux process data, including commands executed and output generated during shell sessions.
```
Comment (norrietaylor) [inline docs/cloud-native-security/cloud-workload-protection.asciidoc:4]: I took a crack at this sentence:

```
Cloud workload protection helps you monitor and protect your Linux hosts and Kubernetes runtimes. It uses the Elastic Defend integration to capture cloud workload telemetry containing process, file and network activity. In Kubernetes environments, the agent further enriches this telemetry with metadata allowing practitioners to isolate events in their cloud topography. 

This telemetry allows for the automation of the identification of cloud threats with out-of-the-box detection rules and machine learning models. The system can then generate alerts based on these detections, which the user can follow with a remediation or response. 
```

I actually think we should come back to the overall description once the use cases are spelled out. I would recommend @nick-alayil as the person to iterate these use cases.

File: docs/cloud-native-security/cloud-workload-protection.asciidoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+[discrete]
+== Capabilities
```
Comment (norrietaylor) [inline docs/cloud-native-security/cloud-workload-protection.asciidoc:7]: Session-view and the dashboard are two views within CWP. I would not classify them as capabilities. 

We should make sure that tell the user that the CWP capabilities are really integrated with the entire Endpoint Defend product.  

- file, network, process telemetry 
- on-host detections and protections such as memory, malicious behaviour, malware. 
- Kibana based rules and ML 
- alert triage
- threat investigation
- Response actions and remediation 

Perhaps we can link to other places in the docs for these topics?

**Review by joepeeples — COMMENTED** [review]
The "Cloud native security" topic is a a great overview, and a fantastic example of the kind of section overviews that we're trying to include in other sections, too. Just a few line edits and format suggestions, thanks!

**Comment by joepeeples** [comment]
(I just added `v8.6.0` to make us this gets backported now that we have the `8.6` branch)

File: docs/cloud-native-security/cloud-workload-protection.asciidoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
+[[cloud-workload-protection]]
+= Cloud workload protection
+
+Cloud workload protection helps you monitor and protect your Linux hosts and Kubernetes runtimes. It uses the <<install-endpoint,{elastic-defend}>> integration to capture cloud workload telemetry containing process, file, and network activity. In Kubernetes environments, this telemetry is further enriched with metadata that enables you to isolate events in your cloud topography.
+
+This telemetry also enables the automated identification of cloud threats with out-of-the-box detection rules and machine learning models. Alerts based on these detections can reduce your mean time to identify and remediate threats.
```
Comment (jmikell821) [inline docs/cloud-native-security/cloud-workload-protection.asciidoc:6]: I'm on the fence about `mean time` here, esp. for our non-native English speakers. Could also be misconstrued for `meantime`. Can we omit `mean` here? If we have to use it, I'd replace with `average`. 
```suggestion
This telemetry also enables the automated identification of cloud threats with out-of-the-box detection rules and machine learning models. Alerts based on these detections can reduce the time to identify and remediate threats.
```

File: docs/cloud-native-security/cloud-workload-protection.asciidoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+[discrete]
+== Use cases
+
+* **Runtime monitoring of cloud workloads:** Provides visibility into cloud workloads, context for detected threats, and the historical data needed for retroactive threat investigations.
+* **Cloud-native threat detection and prevention:** Provides security coverage for Linux, containers, Kubernetes, and serverless applications. Protects from known and unknown threats using on-host detections and protections against malicious behavior, memory threats, and malware.
+* **Reducing the mean time to detect and remediate runtime threats:** Helps you resolve potential threats by showing alerts in context, making the data necessary for further investigations readily available, and providing remediation options.
```
Comment (jmikell821) [inline docs/cloud-native-security/cloud-workload-protection.asciidoc:13]:
```suggestion
* **Reducing the time to detect and remediate runtime threats:** Helps you resolve potential threats by showing alerts in context, making the data necessary for further investigations readily available, and providing remediation options.
```

**Pull request merged**